.github/workflows/ci.yml

```diff
@@ -229,6 +229,7 @@ jobs:
           export PATH=$(python3 -m site --user-base)/bin:${HOME}/.local/bin:${PATH}
           if [ "$ASAN" == "true" ]; then
             # Work-around ASAN bug https://github.com/google/sanitizers/issues/1716
+            sudo sysctl -w kernel.randomize_va_space=0
```

Comment:
Did I miss the discussion? Why is this necessary? Disabling ASLR doesn't seem a good idea to me.